ddtrace/appsec/sqreen.py

```diff
@@ -0,0 +1,33 @@
+from collections import Counter
+import logging
+from typing import Any
+from typing import Mapping
+
+from sq_native import waf  # type: ignore
+
+from ddtrace.appsec.protections import BaseProtection
+
+
+log = logging.getLogger(__name__)
+
+
+class SqreenLibrary(BaseProtection):
+    """
+    Process application data with the Sqreen library.
+    """
+
+    budget = 5 * 1000  # max 5ms budget
```

Comment:
```suggestion
    budget_ms = 5 * 1000  # max 5ms budget
```

might be a tad more clear